src/Microsoft.AspNetCore.WebSockets/Microsoft.AspNetCore.WebSockets.csproj

```diff
@@ -14,6 +14,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="System.Numerics.Vectors" Version="$(CoreFxVersion)" />
```

Comment:
It was removed as part of https://github.com/aspnet/WebSockets/commit/67322fe8aba761c0e0d8c387984899b6458ea0ba